VideoWeb/VideoWeb/ClientApp/src/app/security/auth.guard.ts

```diff
@@ -2,15 +2,21 @@ import { Injectable } from '@angular/core';
 import { ActivatedRouteSnapshot, CanActivate, Router, RouterStateSnapshot } from '@angular/router';
 import { Observable } from 'rxjs';
 import { map } from 'rxjs/operators';
-import { OidcSecurityService } from 'angular-auth-oidc-client';
 import { pageUrls } from '../shared/page-url.constants';
+import { SecurityServiceProvider } from './authentication/security-provider.service';
+import { ISecurityService } from './authentication/security-service.interface';
 
 @Injectable()
 export class AuthGuard implements CanActivate {
-    constructor(private oidcSecurityService: OidcSecurityService, private router: Router) {}
+    private securityService: ISecurityService;
+    constructor(securityServiceProviderService: SecurityServiceProvider, private router: Router) {
+        securityServiceProviderService.currentSecurityService$.subscribe(securityService => {
+            this.securityService = securityService;
+        });
+    }
 
     canActivate(route: ActivatedRouteSnapshot, state: RouterStateSnapshot): Observable<boolean> {
-        return this.oidcSecurityService.isAuthenticated$.pipe(
+        return this.securityService.isAuthenticated$.pipe(
             map((isAuthorized: boolean) => {
                 console.log('AuthorizationGuard, canActivate isAuthorized: ' + isAuthorized);
```

Comment:
replace with logger?

Reply:
Changing this required a change of where the log adapters are setup. They are now setup in the shared module.